ui/apps/platform/src/Containers/NetworkGraph/components/CIDRFormRow.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Field } from 'formik';
+import { Button, Flex, FlexItem, FormGroup } from '@patternfly/react-core';
+import { TrashIcon } from '@patternfly/react-icons';
+
+const CIDRFormRow = ({ idx, onRemoveRow, errors, touched }) => {
+    const { name: nameError, cidr: cidrError } = errors?.entity || {};
+    const { name: nameTouched, cidr: cidrTouched } = touched?.entity || {};
+    const showNameError = nameError && nameTouched;
+    const showCidrError = cidrError && cidrTouched;
+    const hasError = showNameError || showCidrError;
+    let buttonClassName = '';
+    if (idx === 0) {
+        buttonClassName = !hasError
+            ? 'pf-m-align-self-flex-end pf-u-mb-sm'
+            : 'pf-m-align-self-center';
+    }
```

Comment:
this is so that the icon doesn't look awkwardly vertically placed depending on whether there is a header in the form or an error message or not